tests/python/test_matrix.py

```diff
@@ -260,3 +259,23 @@ def to_dest(a):
         [1.0, 2.0, 3.0, 4.0]
     ])
     assert(m == Matrix44f(to_dest(m)))
+
+@pytest.mark.parametrize("package", [ek.scalar])
+def test15_quat_to_euler(package):
+    Quaternion4f, Array3f, Float = package.Quaternion4f, package.Array3f, package.Float
+    prepare(package)
+
+    # Gimbal lock at +pi/2
+    q = Quaternion4f(0, 1/np.sqrt(2), 0, 1/np.sqrt(2))
```

Comment:
```suggestion
    q = Quaternion4f(0.0, 1.0 / np.sqrt(2), 0.0, 1.0 / np.sqrt(2))
```

And please do the same for the rest of this function